src/lobby/controller.js

```diff
@@ -122,9 +117,7 @@ function LobbyController(CardshifterServerAPI, $scope, $timeout, $rootScope, $lo
             CardshifterServerAPI.sendMessage(getCards);
             $location.path("/deck_builder");
         } else {
-            var message = new CardshifterServerAPI.messageTypes.ChatMessage(
-                        "Client error: Select a game type before you can open the deck builder.");
-            addChatMessage(message);
+            ErrorCreator.create("Client error: Select a game type before you can open the deck builder.");
```

Comment:
This message is inconsistent with the other messages. Why does it start with "Client error:"?